rita/src/rita_common/port_emissary/mod.rs

```diff
@@ -0,0 +1,45 @@
+use std::fs::File;
+use std::io::prelude::*;
+use std::u16;
+
+/// Helper function for parsing out port number from local_address column
+fn take_local_port(s: &str) -> u16 {
+    // second column in table contains local_address
+    let local_addr = s.split_whitespace().nth(1).unwrap();
```

Comment:
too many unwraps in this, what if you get invalid input? needs error handling and to return a result type so that we can do something other than crash the program when it goes wrong.